.github/dependabot.yml

```diff
@@ -4,6 +4,22 @@ updates:
     directory: '/'
     schedule:
       interval: 'daily'
+    groups:
+      storybook:
+        patterns:
+          - "@storybook/*"
+      google-cloud:
+        patterns:
+          - "@google-cloud/*"
```

Comment:
This might be casting the net a little too wide.  It's possible that we want to upgrade @google-cloud/tasks without touching @google-cloud/firestore.

Reply:
Yep, we have a separate resolution to control the protobuf that they don't always match on.